src/test/kotlin/no/nav/syfo/testhelper/ExternalMockEnvironment.kt

```diff
@@ -49,6 +50,7 @@ class ExternalMockEnvironment private constructor() {
         pdlUrl = pdlMock.url,
     )
     lateinit var redisCache: RedisStore
+    val redisServer = testRedis(environment)
```

Comment:
Kom i skade for å slette denne i forrige PR - burde ikke gjort det, så legger den tilbake nå (blir en million connection refused mot Redis-cachen uten denne).